matplotcheck/raster.py

```diff
@@ -74,22 +75,18 @@ def _which_label(self, label, all_label_options):
         string that is the first entry in the internal list which label is
         matched with. If no match is found, return value is None
```

Comment:
this just seems confusing. why not just return a boolean list and clarify what this helper does.